libcalico-go/lib/backend/syncersv1/updateprocessors/profileprocessor.go

```diff
@@ -77,25 +76,18 @@ func (pup *profileUpdateProcessor) Process(kvp *model.KVPair) ([]*model.KVPair,
 		}
 	}
 
```

Comment:
Theses changes mean an up level Typha will stop syncing the old v1 Labels key/values.

But older versions of Felix still use the v1 Labels key/values for label inheritance.

Reply:
Addressed by #5695